### PR TITLE
[docs][ci] Deploy using github action

### PR DESCRIPTION
**Pull request by nicknovitski**
After this is merged, pushes to master which touch the docs directory will cause a build and deploy, and pull requests off master which touch the docs directory will cause a build.

# Test Plan

At first I wrote the workflow to run on every push, so I've been watching this branch build successfully.

If there are problems with the deployment specifically, we can revert, and I'll try to fix the problem in another PR

### CODE REVIEW

File: .github/workflows/docs.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  run:
+    shell: bash
+    working-directory: docs
+
+on:
+  push:
+    branches: [ master ]
+    paths:
+      - 'docs/**'
+      - '.github/workflows/docs.yml'
```
Comment (ide) [inline .github/workflows/docs.yml:13]: Is this not implicit? (I am not sure either way, but would like to verify so as not to spread this pattern if it is unnecessary.)

File: .github/workflows/docs.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+          restore-keys: |
+            ${{ runner.os }}-yarn-
+      - run: yarn install --frozen-lockfile
+      - run: yarn danger ci
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - run: yarn export
+      - name: test links
+        run: |
+          yarn run export-server &
```
Comment (ide) [inline .github/workflows/docs.yml:44]: yarn asks not to have "run":
```suggestion
          yarn export-server &
```

File: .github/workflows/docs.yml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+          while ! nc -z localhost 8000; do
+            sleep 1
+          done
+          yarn run test-links http://127.0.0.1:8000
```
Comment (ide) [inline .github/workflows/docs.yml:48]:
```suggestion
          yarn test-links http://127.0.0.1:8000
```

**Review by ide — APPROVED** [review]

File: .github/workflows/docs.yml (hunk 1 above)
Comment (nicknovitski) [reply .github/workflows/docs.yml:13]: It is required.  My first commits only included the docs path and thus _did not_ trigger the workflow until I added this line.

**Pull request merged**